locales/puppet.pot

```diff
@@ -0,0 +1,148 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) 2016 Puppet, LLC.
+# This file is distributed under the same license as the Puppet package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, 2016.
```

Comment:
This file is auto-generated, and as far as I can tell after some searching, there's no way to set this from the configuration file (config.yaml). I believe the intent is to leave a placeholder for the translator to add a title/copyright when generating the .po files from this .pot.